.dockerignore

```diff
@@ -0,0 +1,4 @@
+.env
+.github
+.vscode
+.pytest
```

Comment:
We can exclude a few more things here, like `.git`, `.venv` and `**/__pycache__/`